docs/install.md

```diff
@@ -427,6 +427,13 @@ features](#alpha-features) to be used.
   field contains metadata about resources used in the TaskRun/PipelineRun such as the 
   source from where a remote Task/Pipeline definition was fetched.
 
+- `custom-task-version`: set this flag to the default of "v1alpha1" to have `PipelineRun`s 
+  create `Run`s, or set this flag to "v1beta1" to switch custom tasks 
+  created by PipelineRuns from the default of creating `Run`s to 
+  instead creating the beta `CustomRun`s. This may be needed if you are
+  currently using custom tasks which have already been updated to listen for 
+  `*v1beta1.CustomRun` instead of `*v1alpha1.Run`.
+
```

Comment:
tried to clarify the documentation for the feature flag:

```suggestion
- `custom-task-version`: set this flag to "v1alpha1" to have `PipelineRuns` create `Runs` from Custom Tasks. Set it to "v1beta1" to have `PipelineRuns` create the beta `CustomRuns`. This may be needed if you are using Custom Tasks that have been updated to listen for `*v1beta1.CustomRun` instead of `*v1alpha1.Run`. For more information, see [Runs](runs.md) and [CustomRuns](customruns.md). Flag defaults to "v1alpha1". 

```